public-ui/src/app/views/pm/pm.component.ts

```diff
@@ -48,6 +54,10 @@ export class PhysicalMeasurementsComponent implements OnInit, OnDestroy {
     this.loadingStack.push(true);
     this.dbc.getPmGroups().subscribe(results => {
       this.conceptGroups = results;
+      if (this.searchString) {
+        this.conceptGroups = this.conceptGroups.filter(conceptgroup =>
+          conceptgroup.groupName.toLowerCase().includes(this.searchString));
```

Comment:
SHould you also be lowercasing the search string?

Reply:
Done.